### PR TITLE
Travis and testing file and scripts

### PR DESCRIPTION
**Pull request by mohammedmh**
## Travis and testing 

**Add tarvis.yml and  test script and all the package that are necessary to accomplish this process like ( tape , tap-spec , node env , nodemon and so on . )**

#16 #11

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.17.1",
-    "nodemon": "^1.18.11"
+    "nodemon": "^1.18.11",
+    "bcrypt": "^3.0.6",
```
Comment (Fatmasiam) [inline server/package.json]: And Here ?!!

File: package.json [hunk 2]
```diff
@@ -20,6 +21,17 @@
...
+    "concurrently": "^4.1.0",
+    "nodemon": "^1.18.11",
+    "supertest": "^4.0.2",
+    "tap-spec": "^5.0.0",
+    "tape": "^4.10.1"
+  },
+  "dependencies": {
+    "bcrypt": "^3.0.6",
+    "dotenv": "^7.0.0",
+    "i": "^0.3.6",
```
Comment (Fatmasiam) [inline package.json]: what's the benefit of installing  ``` bcrypt``` , ```jsonwebtoken```, ```dotenv```, ``` i``` and ```npm```  packages  when you set up the Travis?!!
And what is ```i``` package ?

**Review by AbdallahAmmar96 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (mohammedmh) [reply package.json]: done

**Review by Fatmasiam — APPROVED** [review]

**Review by naremanhilles — APPROVED** [review]

**Review by Fatmasiam — CHANGES_REQUESTED** [review]
Solve the conflict !!

**Comment by Fatmasiam** [comment]
The branch name !!! (```TT```)

**Pull request closed without merging**